--- gcc-3.3.3.org/configure.in	2004-01-02 15:09:48.000000000 +0100
+++ gcc-3.3.3/configure.in	2004-03-15 13:46:03.000000000 +0100
@@ -1474,7 +1474,7 @@
 # being built; programs in there won't even run.
 if test "${build}" = "${host}" && test -d ${srcdir}/gcc; then
   # Search for pre-installed headers if nothing else fits.
-  FLAGS_FOR_TARGET=$FLAGS_FOR_TARGET' -B$(build_tooldir)/bin/ -B$(build_tooldir)/lib/ -isystem $(build_tooldir)/include'
+  FLAGS_FOR_TARGET=$FLAGS_FOR_TARGET' -B$(build_tooldir)/bin/ -B$(build_tooldir)/lib/ -isystem $(build_tooldir)/include -isystem /usr/include'
 fi
 
 if test "x${use_gnu_ld}" = x &&
